code/game/objects/structures/transit_tubes/transit_tube_pod.dm

```diff
@@ -34,8 +34,11 @@
 	air_contents.temperature = T20C
 
 	// Give auto tubes time to align before trying to start moving
```

Comment:
might as well kill this comment since there's no more 'spawn' and thus no more giving tubes time to do stuff